src/components/Menus/DropdownMenu.js

```diff
@@ -0,0 +1,211 @@
+import React     from 'react'
+import PropTypes from 'prop-types'
+import Radium    from 'radium'
+import Menu      from './Menu'
+import zIndex    from '../../styles/zIndex'
+
+const styles = {
+  menuContainer: {
+    zIndex: zIndex.Z_INDEX9,
+    position: 'absolute',
+    display: 'none',
+    width: '100%'
+  },
+  menuContainerOpen: {
+    display: 'block'
+  }
+}
+
+@Radium
+class DropdownMenu extends React.Component {
+  static propTypes = {
+    /** MenuItems or Divider */
+    children: PropTypes.node.isRequired,
+    /** React component or HTML that will be used as the dropdown trigger. ie. Button or Icon */
+    triggerElement: PropTypes.element,
+    /** Open and close menu manually */
+    open: PropTypes.bool,
+    /** Callback fired on dropdown close */
+    onClose: PropTypes.func,
+    /** Callback fired on click */
+    onClick: PropTypes.func,
+    /** Callback fired on dropdown open */
+    onOpen: PropTypes.func,
+    /** Callback function fired on key down */
+    onKeyDown: PropTypes.func,
+    /** Requesting to open/close for controlled open component. */
+    onRequestChange: PropTypes.func,
+    /** Callback function fired when a MenuItem is selected */
+    onSelect: PropTypes.func,
+    /** Customize style root element */
+    style: PropTypes.shape({}),
+    /** Customize style of menu parent */
+    menuContainerStyle: PropTypes.shape({}),
+    /** Props passed to Menu component */
+    menuProps: PropTypes.shape({})
+  }
+
+  static defaultProps = {
+    open: null
+  }
+
+  state = {
+    open: false
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.open !== null && this.props.open !== nextProps.open) {
```

Comment:
This can be reduced to `if (nextProps.open && this.props.open !== nextProps.open) {`